docs/02-get-started/README.md

```diff
@@ -24,7 +24,7 @@ All the steps are performed in the `default` Namespace.
   
     Alternatively, you can just use `curl` to publish events.
 - Istio sidecar injection enabled in the `default` Namespace
-  >**NOTE:** Read about [Istio sidecars in Kyma and why you want them](https://kyma-project.io/#/istio/user/00-overview/00-30-overview-istio-sidecars). Then, check how to [enable automatic Istio sidecar proxy injection](https://kyma-project.io/#/istio/user/02-operation-guides/operations/02-20-enable-sidecar-injection). For more details, see [Default Istio setup in Kyma](https://kyma-project.io/#/istio/user/00-overview/00-40-overview-istio-setup).
+  >**NOTE:** Read about [Istio sidecars in Kyma and why you want them](https://kyma-project.io/#/istio/user/00-30-overview-istio-sidecars). Then, check how to [enable automatic Istio sidecar proxy injection](https://kyma-project.io/#/istio/user/operation-guides/02-20-enable-sidecar-injection). For more details, see [Default Istio setup in Kyma](https://kyma-project.io/#/istio/user/00-40-overview-istio-setup).
```

Comment:
Both links are broken, but I guess they will be fixed when this is merged, correct?

Reply:
wow you're fast @grego952  :D

Reply:
Yes, the links will work when I merge the PR to Istio